From: Martin Kepplinger <martink@posteo.de>
Subject: init: fix removal warning for deprecated initrd loading
Date: Wed, 18 Dec 2024 12:36:38 +0000

This won't be removed in 2021, no matter how hard we try.

Link: https://lkml.kernel.org/r/20241218123638.34907-1-martink@posteo.de
Signed-off-by: Martin Kepplinger <martink@posteo.de>
Cc: Christoph Hellwig <hch@lst.de>
Cc: Joel Granados <joel.granados@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 init/do_mounts_initrd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/init/do_mounts_initrd.c~init-fix-removal-warning-for-deprecated-initrd-loading
+++ a/init/do_mounts_initrd.c
@@ -89,7 +89,7 @@ static void __init handle_initrd(char *r
 	extern char *envp_init[];
 	int error;
 
-	pr_warn("using deprecated initrd support, will be removed in 2021.\n");
+	pr_warn("using deprecated initrd support, will be removed soon.\n");
 
 	real_root_dev = new_encode_dev(ROOT_DEV);
 	create_dev("/dev/root.old", Root_RAM0);
_
